site/layouts/shortcodes/include.html

```diff
@@ -1,6 +1,6 @@
 {{ $file := .Get 0 }}
 [{{$file}}]({{printf "/examples/%s" $file}})
-{{ $path := (printf "../../static/examples/%s" $file)}}
+{{ $path := (printf "/opt/build/repo/site/static/examples/%s" $file)}}
```

Comment:
uhm...  this is worrisome. It will work in the bot, but it won't work if someone is trying to build the site locally.

Can you try `"site/static/examples/%"` or `"static/examples/%"`?

Reply:
It sounds reasonable. Let me try it.